pyne/openmc.py

```diff
@@ -156,3 +165,241 @@ def xml(self):
         ace_tables = "\n  ".join([a.xml() for a in self.ace_tables])
         s = template.format(filetype=self.filetype, ace_tables=ace_tables)
         return s
+
+def get_ebins_from_openmc_sp(filename, tally_num):
```

Comment:
Why duplicate the OpenMC python interface? Why not just use it?

Reply:
This PR relates to read and write hdf5 files. I do not use the OpenMC python interface because:
- HDF5 dependencies issue:
    - HDF5 related packages: PyNE uses `pytables` to deal with h5 files, but OpenMC uses `h5py`. Use OpenMC python interface means we need add `h5py` as another dependency.
    - HDF5 file interface: Both PyNE and OpenMC have their own h5 interface. PyNE has `src/h5wrap`. OpenMC has `src/hdf5_interface`. They use different  interfaces.
 - Code dependency: PyNE provides interfaces for different codes by reading their output and writing their input files. But PyNE does not depend on them, i.e., we don't need to install OpenMC if we want to use PyNE OpenMC interface. what we need is the output files of OpenMC. We can install OpenMC and PyNE on different machines.

Reply:
@makeclean Now  I totally agree with you to use the OpenMC python API. I need more and more functions related to OpenMC, it's stupid to do it again. I opened an issue here: #1166 .